src/ts/background/services/account-balance.ts

```diff
@@ -0,0 +1,30 @@
+import t from '../../services/i18n'
```

Comment:
are we planning to call this via message?  if not, we probably can keep it in services rather than in background.